main.go

```diff
@@ -48,16 +50,16 @@ func handler(c net.Conn) {
 
 	q, err := randomQuote()
 	if err != nil {
-		fmt.Println(err)
+		log.Println(err)
 		return
 	}
 	c.Write(q)
 }
 
-
 func main() {
 	a := os.Args[1:]
 	if len(a) < 2 {
+		fmt.Println("Incorrect arguments. The correct order is");
```

Comment:
Why did you use `fmt.Println` here and not `log.Println`?
Also this message does not make sense, as only length is checked, but not order of arguments (which could be done as well).

Reply:
I used fmt.Println instead of log.Println because the user gets notified about the error of the order and /or length of the arguments provided while starting the server. this message makes complete sense when there is an error in the arguments.

Reply:
this error message would actually assist the fmt.Println("usage: port,file") statement. if the arguments are incorrect it would read as

Incorrect arguments, the correct order is 
usage:port, file